TreeTracker/UI/Views/AddTree/AddTreeViewController.swift

```diff
@@ -81,11 +108,27 @@ class AddTreeViewController: UIViewController, AlertPresenting {
             title = viewModel?.title
         }
     }
+    let session = AVCaptureSession()
+    var captureDevice: AVCaptureDevice!
+    var videoDataOutput: AVCaptureVideoDataOutput!
+    var videoDataOutputQueue: DispatchQueue!
+    var previewLayer: AVCaptureVideoPreviewLayer!
+    var cameraView: UIView!
+    let photoOutput = AVCapturePhotoOutput()
```

Comment:
[suggestion:](https://conventionalcomments.org/#labels) Think all these can be private so they can't be accessed from outside the ViewController.